packages/ui/src/theme/css/component/fieldGroup.scss

```diff
@@ -94,6 +104,10 @@
 .amplify-field-group__field-wrapper {
   position: relative;
   width: 100%;
+
+  &[data-orientation='vertical'] {
+    width: fit-content;
```

Comment:
Can you confirm this is working across browsers? I don't think we are auto-prefixing our scss, but I may be wrong. Looks like FF requires prefix like `-moz-fit-content` https://caniuse.com/mdn-css_properties_width_fit-content

Reply:
We do auto-prefix our scss. <- https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/scripts/generateCSS.js#L9